markdown/how.md

```diff
@@ -3,17 +3,62 @@
 
 ## OpenStack
```

Comment:
@fghaas What should we mention here?  
That we use OpenStack Heat to create interactive lab environments for our learning management system.

Reply:
That all of Cleura Cloud runs on OpenStack, and that we use it for both the infrastructure of the learning platform, and for our lab environments.